packages/flutter_test/lib/src/window.dart

```diff
@@ -601,6 +577,39 @@ class FakeAccessibilityFeatures implements ui.AccessibilityFeatures {
   }
 }
 
+/// Used to fake insets and padding for [TestFlutterView]s.
+///
+/// See also:
+///
+///   * [TestFlutterView.padding], [TestFlutterView.systemGestureInsets],
+///     [TestFlutterView.viewInsets], and [TestFlutterView.viewPadding] for test
+///     properties that make use of [FakeViewPadding].
+///   * DEPRECATED: [TestWindow.viewInsetsTestValue], [TestWindow.viewPaddingTestValue],
+///     [TestWindow.paddingTestValue], [TestWindow.systemGestureInsetsTestValue],
+///     for test value setters that can make use of [FakeViewPadding].
```

Comment:
I put it in because it'll resolve [120864](https://github.com/flutter/flutter/issues/120864), which _does_ specify `TestWindow`. I went back and forth on it though, so I'll remove it if you'd like?

Reply:
I am leaning towards removing it. I don't really like advertising APIs that we are about to rip out... But either way is fine with me.

Reply:
Aight. I'll pull it out. I added to the docs for the soon-to-be deprecated methods, so they'll at least have _some_ guidance to use this class.

Reply:
Done.